db/versions/0023-downgrade.sql

```diff
@@ -0,0 +1,30 @@
+begin
+  -- lock this table before reading from it, to prevent loss of concurrent
+  -- updates when the table is dropped.
+  lock table github_access_tokens;
+
+  create table github_access_token_table_entities(
+    partition_key text,
+    row_key text,
+    value jsonb not null,
+    version integer not null,
+    etag uuid default public.gen_random_uuid());
+  alter table github_access_token_table_entities add primary key (partition_key, row_key);
+  grant select, insert, update, delete on github_access_token_table_entities to $db_user_prefix$_web_server;
+
+  insert into github_access_token_table_entities
+  select
```

Comment:
This table is signed, and on downgrade we will have lost the signatures, so I think we need to just drop the data when downgrading.  That will be the case for all signed tables :(

Reply:
Done!